validator/delphi_validator/static.py

```diff
@@ -204,14 +209,16 @@ def find_all_unexpected_geo_ids(df_to_test, geo_regex, geo_type):
                 df_to_test['geo_id']) if geo not in expected_geos}
 
             if len(unexpected_geos) > 0:
-                report.add_raised_error(ValidationError(
-                    ("check_geo_id_format", nameformat),
-                    unexpected_geos, "Non-conforming geo_ids found"))
+                report.add_raised_error(
+                    ValidationFailure("check_geo_id_format",
+                                      nameformat,
+                                      f"Non-conforming geo_ids {unexpected_geos} found"))
 
         if geo_type not in GEO_REGEX_DICT:
```

Comment:
There's a few places in this file that use a "if not... else" pattern which I find harder to read. I would prefer to switch the order of branches.

Not introduced by you, so feel free to skip.

Reply:
No this is an antipattern I have been trying to correct.  Done